src/pages/Services.tsx

```
@@ -0,0 +1,28 @@
+import React from 'react'
+import Navbar from '../components/Navbar'
+import NewsLetter from '../components/NewsLetter';
+import Footer from '../components/Footer';
+import Portfolio from '../components/Portfolio';
+import TopBrands from '../components/TopBrands';
+import CreativeSolution from '../components/CreativeSolution';
+import Map from '../components/WorldMap';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)